_posts/0000-01-02-samuelaparker.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This should help: 
![image](https://user-images.githubusercontent.com/58891435/95203908-603abc80-07b1-11eb-8f73-ea2e9b4f4e53.png)